index.html

```diff
@@ -1158,16 +1162,18 @@ <h3>Credential Subject</h3>
 
         <p>
 A <a>verifiable credential</a> contains <a>claims</a> about a <a>subject</a>.
-The <a>subject</a> of the credential MUST be expressed using the
-<code>credentialSubject</code> <a>property</a>.
+This specification defines a <code>credentialSubject</code> <a>property</a>
+for the expression of <a>claims</a> about a <a>subject</a>. A
+<a>verifiable credential</a> MUST have a <code>credentialSubject</code>
+<a>property</a>.
         </p>
 
         <dl>
           <dt>credentialSubject</dt>
           <dd>
-The value of the <code>credentialSubject</code> <a>property</a> MUST be
-an object that contains one or more properties that are related to the
-<a>subject</a> of the <a>verifiable credential</a>. The object MAY contain an
+The value of the <code>credentialSubject</code> <a>property</a> is defined as
+a set of objects that contain one or more properties that are each related to a
+<a>subject</a> of the <a>verifiable credential</a>. Each object MAY contain an
```

Comment:
I don't believe this is correct.  My suggestion: "The value of the <code>credentialSubject</code> <a>property</a> MUST be an object containing one or more properties.  This specification defines these properties to be related to the <a>subject</a> of the <a>verifiable credential</a>.  The object MAY"

Reply:
Your suggestion would break the spec. :) -- `credentialSubject` can be an array, or more specifically, a set.

Reply:
Oops, forget what I said.